tool/src/main/java/org/wildfly/security/tool/FileSystemEncryptRealmCommand.java

```diff
@@ -0,0 +1,737 @@
+            return hashEncoding;
+        }
+
+        public void setHashEncoding(Encoding hashEncoding) {
+            this.hashEncoding = hashEncoding;
+        }
+
+        public int getLevels() {
+            return levels;
+        }
+
+        public void setLevels(int levels) {
+            this.levels = levels;
+        }
+
+        public String getInputRealmLocation() {
+            return inputRealmLocation;
+        }
+
+        public void setInputRealmLocation(String inputRealmLocation) {
+            this.inputRealmLocation = inputRealmLocation;
+        }
+
+        public String getOutputRealmLocation() {
+            return outputRealmLocation;
+        }
+
+        public void setOutputRealmLocation(String outputRealmLocation) {
+            this.outputRealmLocation = outputRealmLocation;
+        }
+
+        public String getFileSystemRealmName() {
+            return FileSystemRealmName;
+        }
+
+        public void setFileSystemRealmName(String FileSystemRealmName) {
+            this.FileSystemRealmName = FileSystemRealmName;
+        }
+
+        public Boolean getEncoded() {
+            return encoded;
+        }
+
+        public void setEncoded(boolean encoded) {
+            this.encoded = encoded;
+        }
+
+        String getCredentialStore() {
+            return this.credentialStore;
+        }
+
+        String getSecretKeyAlias() {
+            return this.secretKeyAlias;
+        }
+
+        void setCredentialStore(String credentialStore) {
+            this.credentialStore = credentialStore;
+        }
+
+        void setSecretKeyAlias(String secretKeyAlias) {
+            this.secretKeyAlias = secretKeyAlias;
+        }
+
+        void reset() {
+            this.inputRealmLocation = null;
+            this.outputRealmLocation = null;
+            this.FileSystemRealmName = null;
+            this.credentialStore = null;
+            this.secretKeyAlias = null;
+            this.hashCharset = null;
+            this.hashEncoding = null;
+            this.encoded = null;
+            this.levels = 0;
+        }
+    }
+
+    @Override
+    public void execute(String[] args) throws Exception {
+        setStatus(GENERAL_CONFIGURATION_ERROR);
+        cmdLine = parser.parse(options, args, false);
+        setEnableDebug(cmdLine.hasOption(DEBUG_PARAM));
+        if (cmdLine.hasOption(HELP_PARAM)) {
+            help();
+            setStatus(ElytronTool.ElytronToolExitStatus_OK);
+            return;
+        }
+        if (cmdLine.hasOption(SILENT_PARAM)) {
+            silentMode = true;
+        }
+        if (cmdLine.hasOption(SUMMARY_PARAM)) {
+            summaryMode = true;
+            summaryString = new StringBuilder();
+            summaryString.append(String.join("", Collections.nCopies(SUMMARY_WIDTH, "-")));
+            summaryString.append(System.getProperty("line.separator"));
+            summaryString.append("Summary for execution of Elytron-Tool command FileSystemEncrypt");
```

Comment:
I think the command name here should be updated

Reply:
Updated